include/os/linux/spl/sys/debug.h

```diff
@@ -120,6 +120,11 @@ void spl_dumpstack(void);
 		    (long long) (_verify3_right));			\
 	} while (0)
 
+/* Workaround possible coverity bug */
+#if defined(DEBUG) && defined(NDEBUG)
+#undef NDEBUG
+#endif
```

Comment:
Where you able to confirm this resolves the issues with the coverity analysis?  It occurs to me if we think this is actually possible we could just make it fatal at build time and fix the bug.

Reply:
> Where you able to confirm this resolves the issues with the coverity analysis?

Unfortunately, I am only supposed to test things that are in the repository since it does not allow testing private branches. If I add this to send a build to coverity, then I just used one of our few uses of coverity per day and I will need to use another to undo that shortly afterward.

> It occurs to me if we think this is actually possible we could just make it fatal at build time and fix the bug.

I like this idea. I will do that and see what happens locally.

Reply:
Nothing happened when I tried that locally. I am closing this. I will revisit the coverity behavior in the future.